components/course-list/index.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) IBM, Corp. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import "@carbon/colors/scss/colors";
+@import "@carbon/layout/scss/layout";
+@import "@carbon/type/scss/styles";
+
+// TODO: Extract to a theme and improve the naming
```

Comment:
@alexao8 I don't know the context about this TODO but I assume that it's something that we will implement once time we have a theme implemented? Just to know 😄

Reply:
Yes, it's a bad copy/paste from the mega menu scss, my bad 😅 I'll correct it now.